.github/workflows/linux_conditional_compilation.yml

```
@@ -90,7 +90,7 @@ jobs:
 
   Build:
     needs: [Docker, Smart_CI]
-    if: "!needs.smart_ci.outputs.skip_workflow"
+    if: ${{ github.event_name != 'merge_group' && !needs.smart_ci.outputs.skip_workflow }}
```

```suggestion
    if: ${{ github.event_name != 'merge_group' && needs.smart_ci.outputs.skip_workflow != 'True' }}
```

---

It was added to the merge group only because of clang-tidy.
Since clang-tidy is now a separate workflow I have reverted this decision.
I can keep it if necessary.

---

Makes sense, let's use:
```suggestion
    if: ${{ github.event_name != 'merge_group' && needs.smart_ci.outputs.skip_workflow != 'True' }}
```

---

Done